app/home/directives/home-series.component.ts

```diff
@@ -0,0 +1,106 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Observable} from 'rxjs';
+
+import {HalDoc} from '../../shared/cms/haldoc';
+import {StoryModel} from '../../storyedit/models/story.model';
+import {SpinnerComponent} from '../../shared/spinner/spinner.component';
+import {ImageLoaderComponent} from '../../shared/image/image-loader.component';
+import {TimeAgoPipe} from '../../shared/date/timeago.pipe';
+import {HomeStoryComponent} from './home-story.component';
+
+@Component({
+  directives: [SpinnerComponent, ImageLoaderComponent, HomeStoryComponent],
+  pipes: [TimeAgoPipe],
+  selector: 'home-series',
+  styleUrls: ['app/home/directives/home-series.component.css'],
+  template: `
+    <header *ngIf="noseries">
```

Comment:
shouldn't we show some standalone stories even if there are series? just lower down the page maybe?